lib/cast_vote_msg.js

```diff
@@ -1,22 +1,21 @@
 const Assert = require('assert-plus')
 const Vote = require('./vote_entity')
 const Shapes = require('./shapes')
-const { fetchProp } = require('./utils')
-const { ValidationError, NotFoundError } = require('./errors')
+const { NotFoundError } = require('./errors')
```

Comment:
Use the module.exports.errors = {} pattern for errors
see https://github.com/senecajs/seneca-cache/blob/master/cache.js for example usage

Reply:
@rjrodger But then how can a client using this plugin, handle the error, if the whole process just crashes?